1-exercises/G-array-methods/exercise.js

```diff
@@ -4,7 +4,7 @@
 */
 
 let numbers = [3, 2, 1];
-let sortedNumbers; // complete this statement
+let sortedNumbers = numbers.sort((a, b) => a - b); // complete this statement
```

Comment:
The function you're passed as an arguement to .sort() is uneccessary here - it still works but it complicates matters, sort() is usually used without an arguement